app/styles/component/_site-head.scss

```diff
@@ -11,6 +11,11 @@ $site-head-default-box-shadow: 0 1px 4px 0 $site-head-box-shadow-color;
 
 .site-head-headroom-un-pinned .site-head {
 	box-shadow: $site-head-default-box-shadow;
+
+	&.no-shadow {
+		box-shadow: 0 0 0 0 $site-head-box-shadow-color;
+		transition: box-shadow .3s;
```

Comment:
animiating a box-shadow is slow, is it needed here?

or can we optimize it?
http://tobiasahlin.com/blog/how-to-animate-box-shadow/